### PR TITLE
Zigbee samples improvements

### PR DESCRIPTION
**Pull request by sebastiandraus**
- Light Bulb sample to use its default LED for identifying
- Network coordinator - add missing docs

### CODE REVIEW

**Comment by sebastiandraus** [comment]
@tomchy What do you think about adding visual effect to Light Bulb sample for Identifying Mode?
This is to align with other samples (Network Coordinator, Template, Shell) that already have visual efect for that.

File: samples/zigbee/light_bulb/src/main.c [hunk 1]
```diff
@@ -185,35 +185,42 @@ ZB_HA_DECLARE_DIMMABLE_LIGHT_CTX(
...
+		ZB_ERROR_CHECK(zb_err_code);
+	} else {
+		LOG_INF("Cancel identify mode");
+		zb_bdb_finding_binding_target_cancel();
+	}
+}
+
+/**@brief Callback for button events.
+ *
+ * @param[in]   button_state  Bitmask containing buttons state.
```
Comment (kittydepa) [inline samples/zigbee/light_bulb/src/main.c:214]: `Bitmask containing state of the buttons` if plural, 
or `Bitmask containing the button's` if singular.

File: samples/zigbee/light_bulb/src/main.c [hunk 2]
```diff
@@ -329,6 +336,42 @@ static void on_off_set_value(zb_bool_t on)
 	}
 }
 
+/**@brief Function to toggle the identify LED - BULB_LED is used for this.
```
Comment (kittydepa) [inline samples/zigbee/light_bulb/src/main.c:339]: Should this be "... the identifying LED", or "...the Identify LED" perhaps?

**Review by kittydepa — APPROVED** [review]

**Review by tomchy — APPROVED** [review]

File: samples/zigbee/light_bulb/src/main.c (hunk 1 above)
Comment (sebastiandraus) [reply samples/zigbee/light_bulb/src/main.c]: Changed to your suggestion

**Comment by sebastiandraus** [comment]
Force pushed to retrigger CI

**Pull request merged**